client/src/main/java/org/eclipse/hono/client/impl/AsyncCommandClientImpl.java

```diff
@@ -117,7 +117,7 @@ static String getLinkTargetAddress(final String tenantId) {
         MessageHelper.setPayload(message, contentType, data);
         message.setSubject(command);
         message.setAddress(targetAddress);
-        final String replyToAddress = String.format("%s/%s/%s", CommandConstants.COMMAND_ENDPOINT, tenantId, replyId);
+        final String replyToAddress = String.format("%s/%s/%s", CommandConstants.NORTHBOUND_COMMAND_RESPONSE_ENDPOINT, tenantId, replyId);
```

Comment:
IMHO we should use *getEndpoint* here instead of the constant, shouldn't we?

Reply:
*getEndpoint()* would return the *request* endpoint. However, in the above line we need the *response* endpoint.